types/cesium/index.d.ts

```diff
@@ -1485,7 +1485,7 @@ declare namespace Cesium {
         origin: Cartesian3;
         direction: Cartesian3;
         constructor(origin?: Cartesian3, direction?: Cartesian3);
-        static getPoint(t: number, result?: Cartesian3): Cartesian3;
+        static getPoint(t: Ray, result?: Cartesian3): Cartesian3;
```

Comment:
should be 

`static getPoint(ray: Ray, t: number, result?: Cartesian3): Cartesian3;`

https://cesium.com/docs/cesiumjs-ref-doc/Ray.html#.getPoint

Reply:
you are right. I've followed https://github.com/AnalyticalGraphicsInc/cesium/blob/1.66/Source/Core/Ray.js#L67 now.